### DISCUSSION TITLE
New Product for matching patient and health care providers

### DISCUSSION THREAD

**Discussion by Paul-B98** [helpwave/helpwave#44] [Hackathon]
The Problem of todays patient is to find a suitable health care offer for there needs. Sometimes they find nothing near there location, which can be a problem if the person has special needs. Another problem is that most of the time there is a waiting list for specific therapies when the patient needs one right now.

This Product should be patient focused with the aim to enable patient to find health care providers they need. This should allow the patient to find the right doctor’s office or therapy offer for their needs in time and the correct location.

**Comment by aegis301** [comment]
Possible competitors:
https://www.kvwl.de/arztsuche
Arztsuche from ärztekammer. Only shows location. Maybe they have an API or a list we could use as basis though. 

https://www.jameda.de/
Does not show waiting time, possibly enables scheduling but so far I saw no one using this.
